### PR TITLE
Gate 1 test cases and how to

### PR DESCRIPTION
**Pull request by wuder4**
This PR has Gate 1 test cases and how to file to help you run the tests.

moving test files from docs to platform

### CODE REVIEW

**Comment by ashakunt** [comment]
Please update the commit message. You and I know that we are moving tests from docs repository but after the commit, nobody knows. So, the commit should say something like, Gate 1 test cases in selenium

File: tests/Gate_1/1.registering.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	Wait Until Page Contains	Required Field
+
+Make new account (pass)
+	Register to apinf	asdas	asd@asd.asd	asdasd
+	confirm page loaded	Users
+
+	
+*** Keywords *** 
+Go to homepage
+	Open Browser	${HOMEPAGE} 	${BROWSER}
```
Comment (ashakunt) [inline tests/Gate_1/1.registering.txt]: Keywords like __Go to homepage__, __Go to login__ and a few others are used in various files. It would be good if we can write all common **utils** in single file and include this file in all the tests. This file should reside outside of the Gate_1 (or Gate_N) folder so that it can be used at any place.

Comment (wuder4) [inline tests/Gate_1/1.registering.txt]: not sure but we may be able to do this with argument file.  I haven't looked into it much yet.
[http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#keyword-arguments](url)

Comment (ashakunt) [inline tests/Gate_1/1.registering.txt]: Check this [link](https://stackoverflow.com/questions/32244489/how-can-i-import-a-file-with-common-keywords-in-robot-framework), which explains how this can be done.

**Comment by ashakunt** [comment]
@wuder4 : also, squash these 3 commits into one single commit as they are redundant.

**Comment by ashakunt** [comment]
Some more suggested changes: 
* Rename the .txt files to .robot
* Rename the How_to.txt to README.md

**Comment by ashakunt** [comment]
Thanks, @wuder4. Now this is getting in shape. Next things we should do is:

1. Move common items to a `common.robot` or `utilities.robot` in the tests folder so that it can be used by all `Gates`. 
2. Update the `README.md` about how to run the tests (along with setup), by default, considering that the tests are run on the local development environment.
3. Write a `run-tests.sh` script which is a wrapper around the _test run command_

**Comment by ashakunt** [comment]
@wuder4 : have you removed the chimp js files for a specific reason?

**Comment by ashakunt** [comment]
@wuder4 : also, i do not understand the reasoning behind the commit message `folder access problems`

File: users/collection/server/api.js [hunk 2]
```diff
@@ -5,677 +5,124 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
 
 // Meteor packages imports
 import { Meteor } from 'meteor/meteor';
-import { Accounts } from 'meteor/accounts-base';
```
Comment (marla-singer) [inline users/collection/server/api.js]: @wuder4  Actually, I don't understand why did you touch and edit this file? How is this related to testing? You removed all logic from this file

File: tests/chimp-config.js [hunk 3]
```diff
@@ -1,35 +0,0 @@
-/* Copyright 2017 Apinf Oy
```
Comment (marla-singer) [inline tests/chimp-config.js]: @wuder4 @ashakunt Also I don't understand why are you going to remove all files from `tests` folder which was created by @mauriciovieira ? Was it stipulated?

PR doesn't contain any reference lint to issue and manipulation in this PR is confused

File: users/client/account/password/password.html [hunk 4]
```diff
@@ -12,11 +12,11 @@
     <legend>
       {{_ 'updatePassword_legend_text' }}
     </legend>
-    {{> afQuickField name="old" type="password" id="password-old" }}
-    {{> afQuickField name="new" type="password" id="password-new" }}
-    {{> afQuickField name="confirm" type="password" id="password-new-again" }}
+    {{> afQuickField name="old" type="password" }}
```
Comment (marla-singer) [inline users/client/account/password/password.html]: @wuder4 Also in the separated merged PR #2655 these set of IDs was added by yourself.
joxi_screenshot_1498207367685

**Comment by wuder4** [comment]
All these changes were made during git rebase command witch also made me unable to access the folder. I'm starting to wonder why we do git rebase since it gives almost 1 hour worth of conflicts to files that are not even modified by you.

**Comment by ashakunt** [comment]
Please note, all the `chimp.js` related files must remain. They should not be deleted.

File: tests/chimp-config.js (hunk 3 above)
Comment (ashakunt) [inline tests/chimp-config.js]: @marla-singer : I am sure those files were removed accidentally. @wuder4  will add them back.

**Comment by ashakunt** [comment]
> All these changes were made during git rebase command witch also made me unable to access the folder. I'm starting to wonder why we do git rebase since it gives almost 1 hour worth of conflicts to files that are not even modified by you.

Possible that the rebase you did was wrong.

**Comment by ashakunt** [comment]
@wuder4 @marla-singer : apparently, I was a bit out of date with the latest commit in this branch. Everything seems to be restored now. Thanks, @wuder4!

**Comment by ashakunt** [comment]
@wuder4: I still find both, How_to.txt and README.md in the `Gate_1` folder. Shouldn't they move to `tests` folder instead and one of them (`How_to.txt`) be deleted?

**Comment by ashakunt** [comment]
@wuder4 : Lets finish this checklist and then, I think this PR is ready to be merged to `develop`: 

* [ ] Re-word the current commit `resetting after rebase conflicts` to have a meaningful context. Your choice of commands: 
        1. `$ git commit --amend` // since this is the last commit in the branch OR
        2. `$ git rebase -i 7ac10ab6e8165abe8ca7c5a058625e5fa0fa01d3` // subsequently, use r to reword

* [ ] Move `README.md` to `tests` folder (`git mv README.md ../README.md`)
* [ ] Delete `How_to.txt` (`git rm How_to.txt`)
